config/openhpc.yml

```diff
@@ -106,7 +106,7 @@ gluster_src: "localhost:/{{ gluster_cluster_volume_name }}"
 gluster_mnt: "/mnt/{{ gluster_cluster_volume_name }}"
 
 # Config for Ceph mount
-ceph_mount_share_name: HomeDirs
+#ceph_mount_share_name: HomeDirs
```

Comment:
Should we just delete this line?

Reply:
Its not in group_vars at the moment.

Reply:
Looks like it controls whether a ceph mount is set up, so if we put it in group_vars it will always be defined. I guess its ok to leave this here to show how to enable the ceph mount. In future it might be clearer to use a separate flag, e.g. ceph_mount_enabled.